app/views/games/index.html.erb

```diff
@@ -6,5 +6,10 @@
   <%= link_to 'show', game_path(game) %>
   <%= link_to 'edit', edit_game_path(game) %>
   <%= link_to 'delete', game, method: :delete %>
+	<%= if @games.available != nil %>
```

Comment:
- Check lines 9-13: remove the '=' unless you are using embedded ruby to render something on the view
- Is there anything you want to do with the information from available games even before we add the functionality to join a game?